### ISSUE TITLE
Bug: Augur "Create A Market" system

### ISSUE THREAD

**Issue by Villuvee** [AugurProject/augur#9550]
I'm trying to create a market on Augur platform for US Presidential Elections 2024. This option is also available in the US Politics market creation system. But when I try to put a market expiration date, then there's no possibility to put any later date than dates in July 2021..How can I create a market for US Presidential Elections 2024 then if I can't set a proper market expiration date for that?

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - Browser: [e.g. chrome, safari]
 - Version: [e.g. 22]

**Smartphone (please complete the following information):**
 - Browser: [e.g. stock browser, safari]
 - Version: [e.g. 22]

**Additional context**

**Comment by Villuvee** [comment]
And if I try to write down the correct date manually (for example January 24th 2025), it says when I try to move forward that "Event expiration time must be in the future".

**Comment by pgebheim** [comment]
In order to facilitate potential upgrades, markets are restricted to having an end date that is before a pre-set Upgrade Day. This allows all markets to resolve so that any necessary token migrations don't interrupt existing unresolved markets.

TLDR; Markets cannot be made that far in the future, they must resolve before next year's upgrade day.

**Comment by Villuvee** [comment]
Do you happen to know is there any other platform which allows longer-term bets and market creations? Or is this universal technical problem? I didn't find information about that before when researched and deposited money only because I wanted to try to create one special market for 2024.
